.github/workflows/ci.yml

```diff
@@ -0,0 +1,71 @@
+name: CI
+
+on: 
+  pull_request:
+    branches:
+      - master
+  push:
+    branches-ignore:
+      - '**'
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
+
+jobs:
+  test-hackatom:
+    name: Rust stable on ${{ matrix.build }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - build: macOS
+            os: macOS-latest
+          - build: Windows
+            os: windows-latest
+    defaults:
+      run:
+        shell: bash
+        working-directory: ./contracts/hackatom
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+      - name: Install rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: 1.47.0
+          target: wasm32-unknown-unknown
+          profile: minimal
+          override: true
+      - name: Build hackatom wasm
+        run: cargo wasm --locked
+      - name: Check hackatom
+        run: cargo check --locked --tests
```

Comment:
Those compile checks should be covered by `cargo wasm --locked` and `cargo unit-test --locked`. Can we remove it?